models/staging/graph/stg_naming_convention_folders.sql

```diff
@@ -0,0 +1,11 @@
+{% set vars_prefix = ['staging_folder_name','intermediate_folder_name','marts_folder_name'] %}
```

Comment:
Do we want to add the ability to add other folders? `other_folder_name`

Reply:
My feeling would be that we start as-is, and mention in the docs what someone would have to do if she/he wanted to add more folders.

(another option would be to create a variable that lists all the different variables (incep-variable-tion) so a person would not have to update the model, they would just have to update the variable of variable, but it doesn't feel like this is a must have for the first release)